packages/web-components/src/components/cbp-toggle/cbp-toggle.stories.tsx

```diff
@@ -81,17 +79,16 @@ function generateToggles(items, labelWidth, hideStatus, statusTexton, statusText
         ${context && context != 'light-inverts' ? `context=${context}` : ''}
         ${sx ? `sx=${JSON.stringify(sx)}` : ``}
         >
-            <label
-                slot='toggleLabel'
+            <span
                 style='width: ${labelWidth}'
             >
                 ${label} 
-            </label>
-            <input
-                type="checkbox"
-                name="checkbox"
-                value="1"
-            />
+           </span>
+                <input
+                    type="checkbox"
+                    name="checkbox"
```

Comment:
you should probably pass the name and value in the object and iterate one of them. A group of checkboxes with the same name and value would not be useful in real life.

Reply:
updated as requested